### ISSUE TITLE
Enable PR pipeline execution when opened

### ISSUE THREAD

**Issue by braisvq1996** [opendevstack/ods-core#1194]
**Is your feature request related to a problem?**
When a Pull Request is opened into a repository of a project the Jenkins pipeline is not executed.
It is necesary to make an aditional commit in order to trigger the jenkins build

**Describe the solution you'd like**
Enable PRs to tigger jenkins pipelines by default

**Describe alternatives you've considered**
Document how users can enable it for their components

**Additional context**
It needs to update the webhook proxy [accepted events](https://github.com/opendevstack/ods-core/blob/master/jenkins/webhook-proxy/main.go#L36) and the Prov-app config map, that takes the value from [ods-configuration](https://github.com/opendevstack/ods-core/blob/master/configuration-sample/ods-core.env.sample#L335-L339)

@metmajer this is what we talked about today

**Comment by braisvq1996** [comment]
I still need to run some tests about this but this is where I think would need to be changed to enable PR execution upon creation

**Comment by tbugfinder** [comment]
it can also be configured using env var
https://github.com/opendevstack/ods-core/blob/master/jenkins/webhook-proxy/main.go#L35

**Comment by braisvq1996** [comment]
Yeah, it can be done there but Bitbucket webhook events also needs to be configured.
